packages/opentelemetry-exporter-collector/src/platform/node/README.md

```diff
@@ -44,3 +44,5 @@ the latest sha when this guide was written is `b54688569186e0b862bf7462a983ccf2c
     ```
 
 9. If you look now at git log you will notice that the folder `protos` has been changed and it will show what was the previous sha and what is current one
+
+Make sure it compiles correctly.
```

Comment:
This is part of previous step or something else. Looks a little disconnected at this moment.